src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,55 @@
 import React from 'react';
 
 import './MovieCard.scss';
+import { Movie } from '../../types/Movie';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+interface Props {
+  movies: Movie[];
+}
+
+export const MovieCard: React.FC<Props> = ({ movies }) => (
+  <>
+    {movies.map((movie) => (
+      <div className="card" data-cy="Movie" key={movie.imdbId}>
+        <div className="card-image">
+          <figure className="image is-4by3">
+            <img
+              data-cy="MovieImage"
+              src={movie.imgUrl}
+              alt="Film logo"
+            />
+          </figure>
+        </div>
```

Comment:
you need to move the map in the MoviesList component  - here you just render movie card